scripts/mkdist.js

```diff
@@ -1,30 +1,30 @@
-var fs = require('fs')
+const fs = require('fs')
+const {join} = require('path')
+const mkdirp = require('mkdirp')
```

Comment:
Again the user does not need to care about this. This is transpiled using babel when using this through gulp, so all is needed is running `npm install` and things work. 

There is really no point in using node without npm modules. Also the site needs to be build after the releases are built, so without node + npm this repo doesn't work anyway. If you really want to drop the nodejs you will have to find someone else to write this as this means increasing the scope of this project quite large and loosing most attempts of having decent cross platform capabilites.